13-transport.Rmd

```diff
@@ -692,17 +690,16 @@ The methods presented in this section are relatively simple compared with what i
 The dual graph/spatial capabilities that **sfnetworks** opens up enable many new powerful techniques can cannot be fully covered in this section.
 This section does, however, provide a strong starting point for further exploration and research into the area.
 A final point is that the example dataset we used above is relatively small.
-It may also be worth considering how the work could adapt to larger networks: testing methods on a subset of the data, and ensuring you have enough RAM will help, although it's also worth exploring other tools that can do transport network analysis that are optimised for large networks, such as R5 [@alessandretti_multimodal_2022].
-
+It may also be worth considering how the work could adapt to larger networks: testing methods on a subset of the data, and ensuring you have enough RAM will help, although it's also worth exploring other tools that can do transport network analysis that are optimized for large networks, such as R5 [@alessandretti_multimodal_2022].
 
 ## Prioritizing new infrastructure
 
-This final practical section demonstrates the policy relevance of geocomputation for transport planning and policies, with a simple example to help prioritise locations new investment.
+This final practical section demonstrates the policy relevance of geocomputation for transport planning and policies, with a simple example to help prioritize locations of a new investment.
```

Comment:
I don't think "a " is needed here: there is usually more than one.

Reply:
So "new investments"?

Reply:
No, investment that is spent in many places. E.g.: 

> The UK government has announced billions of pounds of new investment in roads and charging infrastructure for electric vehicles as part of its Spending Review 2020.

That is better than "new investments in roads" and the same applies to locations of new investment. There's a definite typo fix with the addition of "of" so let's merge it. Happy to make post merge changes.

Source: https://www.eunomia.co.uk/category/policy-strategy/sustainable-future-transport/

Reply:
👍🏻